resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -118,19 +118,19 @@ s3.copy.bucket.action=Copy Name
 s3.copy.bucket.failed=Copy Bucket Failed
 s3.open.viewer.bucket.action=Open Bucket in Viewer
 s3.open.viewer.bucket.failed=Failed to open bucket
-s3.delete.object.action=Delete Object
+s3.delete.object.action=Delete Object...
```

Comment:
I don't like the addition of ellipses, it looks cleaner without them

Reply:
Other than that, looks like a nice UI change!

Reply:
The UI guidelines say to use ellipsis if the user needs to confirm or provide more input:

https://jetbrains.github.io/ui/controls/button/#10

Reply:
Sounds good, we need to edit other parts our UI to follow this (not as part of this PR, obviously)